ariadne/contrib/federation/schema.py

```diff
@@ -52,8 +64,12 @@ def make_federated_schema(
     # Remove custom schema directives (to avoid apollo-gateway crashes).
     # NOTE: This does NOT interfere with ariadne's directives support.
     sdl = purge_schema_directives(type_defs)
+    extend_token = "extend " if has_query_type(sdl) else ""
```

Comment:
I'm thinking if this wouldn't be readable if we inserted `extend type` or `type` into query template.

Reply:
Agreed 👌

Reply:
Done.